be/src/Project/Web/Seller/Seller.Web/Areas/Products/ApiControllers/CategoriesApiController.cs

```diff
@@ -50,7 +51,7 @@ public async Task<IActionResult> Index([FromBody] SaveCategoryRequestModel model
             var language = CultureInfo.CurrentUICulture.Name;
 
             var categoryId = await this.categoriesRepository.SaveAsync(
-                token, language, model.Id, model.ParentCategoryId, model.Name, model.Files.Select(x => x.Id.Value), model.Schema, model.UiSchema);
+                token, language, model.Id, model.ParentCategoryId, model.Name, model.Files.Select(x => x.Id.Value), model.Schemas);
```

Comment:
**suggestion:** Should be:

![obraz](https://github.com/dawiddworak88/Giuru/assets/93117743/f1321cdd-da6e-4942-b2d2-2d51a32f2b08)